net-analyzer/wtfis/wtfis-0.6.2.ebuild

```diff
@@ -15,14 +15,14 @@ if [[ ${PV} = "9999" ]]; then
 	EGIT_REPO_URI="https://github.com/pirxthepilot/wtfis.git"
 else
 	SRC_URI="https://github.com/pirxthepilot/wtfis/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
-	KEYWORDS="amd64 ~loong x86"
+	KEYWORDS="~amd64 ~x86"
```

Comment:
Is this an intentional drop of the loong keyword?

Reply:
For this release we need `>=dev-python/pydantic-2` which is not (yet) keyworded for `loong`. But we can readd it, if `dev-python/pydantic` gets keyworded for `loong` again.